doc/Run_your_personal_case.rst

```diff
@@ -0,0 +1,160 @@
+    Once you have selected the area of interest by moving around the map, press the ‘export’ button. 
+    You should now see a window on the left with a set of co-ordinates. By selecting ‘Select a different area manually’ 
+    you can change the boundaries of the micro-network and find the desired set of co-ordinates to insert into the tool.
+
+To enter the coordinates, simply change the values in the microgrid_list section to the desired values, 
+in our case:
+
+.. code:: yaml
+
+    microgrids_list:
+        your_microgrid:
+            lon_max: 27.29261
+            lon_min: 27.23579
+            lat_min: 37.81941
+            lat_max: 37.88786
+
+
+The code is able to simulate several systems at the same time. 
+To do so, simply add an additional analogous block to your_microgrid. For example:
+.. code:: yaml
+
+    microgrids_list:
+        microgrid_1:
+            lon_max: 27.29261
+            lon_min: 27.23579
+            lat_min: 37.81941
+            lat_max: 37.88786
+
+        microgrid_2:
+            lon_max: 27.3941
+            lon_min: 27.3556
+            lat_min: 37.9334
+            lat_max: 37.9611
+
+Configure ``enable`` section to download/build data
+---------------------------------------------------------
+This section determines how much of the data required for the simulation is to be obtained. 
+A configuration of this type is effective for launching the simulation, the inputs can of course be changed as required. 
+Reference can also be made to the documentation in PyPSA-Earth on this.
+
+.. code:: yaml
+
+    enable:
+        retrieve_databundle: true  # Recommended 'true', for the first run. Otherwise data might be missing.
+        retrieve_cost_data: true   # If true, it retrieves cost data from technology data and saves in resources/costs.csv, if false uses cost data in data/costs.csv
+        download_osm_data: true  # If true, OpenStreetMap data will be downloaded for the selected countries
+        download_osm_buildings: true  # If true, OpenStreetMap buildings will be downloaded for the selected countries
+        download_osm_method: overpass # or earth_osm
+        build_cutout: false
+        build_natura_raster: false  # If True, than an exclusion raster will be build
+
+
+Specify the weather year scope
+------------------------------
+Let us assume, as already mentioned, that we want to run a simulation on an annual horizon. 
+With this operation, the snapshots that will be associated with the network are set.
+
+.. note::
+    Attention: with this operation, the year of the climate data used for the calculation of the renewable yield is not selected, 
+    this information depends on the cutout used. 
+
+.. code:: yaml
+
+    snapshots:
+        start: "2013-01-01"
+        end: "2014-01-01"
+        inclusive: "left" # end is not inclusive
+
+specify the method of load calculation :
+------------------------------
+Currently, energy demand can be simulated with two different strategies. 
```

Comment:
Better to add a phrase here to explain the two methodologies, like
"... strategies: (1) with the RAMP tool and (2) by using static profiles".
Then you explain briefly the options